diff --git a/node_modules/@stencil/core/internal/client/index.js b/node_modules/@stencil/core/internal/client/index.js
index 767b59b..3a861af 100644
--- a/node_modules/@stencil/core/internal/client/index.js
+++ b/node_modules/@stencil/core/internal/client/index.js
@@ -3987,7 +3987,7 @@ const loadModule = (cmpMeta, hostRef, hmrVersionId) => {
 };
 const styles = /*@__PURE__*/ new Map();
 const modeResolutionChain = [];
-const win = typeof window !== 'undefined' ? window : {};
+const win = typeof window !== 'undefined' ? window : globalThis || {};
 const doc = win.document || { head: {} };
 const H = (win.HTMLElement || class {
 });
