diff --git a/node_modules/@stencil/core/mock-doc/index.cjs b/node_modules/@stencil/core/mock-doc/index.cjs
index 08b0e54..f28255e 100644
--- a/node_modules/@stencil/core/mock-doc/index.cjs
+++ b/node_modules/@stencil/core/mock-doc/index.cjs
@@ -6843,16 +6843,10 @@ var MockElement = class extends MockNode2 {
   __shadowRoot;
   __style;
   attachInternals() {
-    return new Proxy({}, {
-      get: function(_target, prop, _receiver) {
-        if ("process" in globalThis && globalThis.process.env.__STENCIL_SPEC_TESTS__) {
-          console.error(
-            `NOTE: Property ${String(prop)} was accessed on ElementInternals, but this property is not implemented.
-  Testing components with ElementInternals is fully supported in e2e tests.`
-          );
-        }
-      }
-    });
+    return {
+      setFormValue: (value) => { },
+      setValidity: (value) => {}
+    };
   }
   constructor(ownerDocument, nodeName, namespaceURI = null) {
     super(ownerDocument, 1 /* ELEMENT_NODE */, typeof nodeName === "string" ? nodeName : null, null);
diff --git a/node_modules/@stencil/core/mock-doc/index.js b/node_modules/@stencil/core/mock-doc/index.js
index dee623c..a64a6c9 100644
--- a/node_modules/@stencil/core/mock-doc/index.js
+++ b/node_modules/@stencil/core/mock-doc/index.js
@@ -6791,16 +6791,10 @@ var MockElement = class extends MockNode2 {
   __shadowRoot;
   __style;
   attachInternals() {
-    return new Proxy({}, {
-      get: function(_target, prop, _receiver) {
-        if ("process" in globalThis && globalThis.process.env.__STENCIL_SPEC_TESTS__) {
-          console.error(
-            `NOTE: Property ${String(prop)} was accessed on ElementInternals, but this property is not implemented.
-  Testing components with ElementInternals is fully supported in e2e tests.`
-          );
-        }
-      }
-    });
+    return {
+      setFormValue: (value) => { },
+      setValidity: (value) => {}
+    };
   }
   constructor(ownerDocument, nodeName, namespaceURI = null) {
     super(ownerDocument, 1 /* ELEMENT_NODE */, typeof nodeName === "string" ? nodeName : null, null);
