diff --git a/node_modules/@stencil/core/mock-doc/index.cjs b/node_modules/@stencil/core/mock-doc/index.cjs
index b8beb06..a78a245 100644
--- a/node_modules/@stencil/core/mock-doc/index.cjs
+++ b/node_modules/@stencil/core/mock-doc/index.cjs
@@ -6807,16 +6807,7 @@ var MockNodeList = class {
 };
 var MockElement = class extends MockNode2 {
   attachInternals() {
-    return new Proxy({}, {
-      get: function(_target, prop, _receiver) {
-        if ("process" in globalThis && globalThis.process.env.__STENCIL_SPEC_TESTS__) {
-          console.error(
-            `NOTE: Property ${String(prop)} was accessed on ElementInternals, but this property is not implemented.
-  Testing components with ElementInternals is fully supported in e2e tests.`
-          );
-        }
-      }
-    });
+    return { setFormValue: (value) => {} };
   }
   constructor(ownerDocument, nodeName, namespaceURI = null) {
     super(ownerDocument, 1 /* ELEMENT_NODE */, typeof nodeName === "string" ? nodeName : null, null);
diff --git a/node_modules/@stencil/core/mock-doc/index.js b/node_modules/@stencil/core/mock-doc/index.js
index dc74c34..5470a34 100644
--- a/node_modules/@stencil/core/mock-doc/index.js
+++ b/node_modules/@stencil/core/mock-doc/index.js
@@ -6754,16 +6754,7 @@ var MockNodeList = class {
 };
 var MockElement = class extends MockNode2 {
   attachInternals() {
-    return new Proxy({}, {
-      get: function(_target, prop, _receiver) {
-        if ("process" in globalThis && globalThis.process.env.__STENCIL_SPEC_TESTS__) {
-          console.error(
-            `NOTE: Property ${String(prop)} was accessed on ElementInternals, but this property is not implemented.
-  Testing components with ElementInternals is fully supported in e2e tests.`
-          );
-        }
-      }
-    });
+    return { setFormValue: (value) => {} };
   }
   constructor(ownerDocument, nodeName, namespaceURI = null) {
     super(ownerDocument, 1 /* ELEMENT_NODE */, typeof nodeName === "string" ? nodeName : null, null);
