diff --git a/node_modules/@stencil/core/internal/client/index.js b/node_modules/@stencil/core/internal/client/index.js
index a5153fa..c7dec25 100644
--- a/node_modules/@stencil/core/internal/client/index.js
+++ b/node_modules/@stencil/core/internal/client/index.js
@@ -4008,7 +4008,7 @@ const loadModule = (cmpMeta, hostRef, hmrVersionId) => {
 };
 const styles = /*@__PURE__*/ new Map();
 const modeResolutionChain = [];
-const win = typeof window !== 'undefined' ? window : {};
+const win = typeof window !== 'undefined' ? window : globalThis || {};
 const doc = win.document || { head: {} };
 const H = (win.HTMLElement || class {
 });
diff --git a/node_modules/@stencil/core/mock-doc/index.cjs b/node_modules/@stencil/core/mock-doc/index.cjs
index d7fcb35..5da16f8 100644
--- a/node_modules/@stencil/core/mock-doc/index.cjs
+++ b/node_modules/@stencil/core/mock-doc/index.cjs
@@ -3306,12 +3306,7 @@ class MockNodeList {
 }
 class MockElement extends MockNode {
     attachInternals() {
-        return new Proxy({}, {
-            get: function (_target, prop, _receiver) {
-                console.error(`NOTE: Property ${String(prop)} was accessed on ElementInternals, but this property is not implemented.
-Testing components with ElementInternals is fully supported in e2e tests.`);
-            },
-        });
+        return { setFormValue: (value) => {}};
     }
     constructor(ownerDocument, nodeName, namespaceURI = null) {
         super(ownerDocument, 1 /* NODE_TYPES.ELEMENT_NODE */, typeof nodeName === 'string' ? nodeName : null, null);
diff --git a/node_modules/@stencil/core/mock-doc/index.js b/node_modules/@stencil/core/mock-doc/index.js
index 0d59979..817e141 100644
--- a/node_modules/@stencil/core/mock-doc/index.js
+++ b/node_modules/@stencil/core/mock-doc/index.js
@@ -3303,12 +3303,7 @@ class MockNodeList {
 }
 class MockElement extends MockNode {
     attachInternals() {
-        return new Proxy({}, {
-            get: function (_target, prop, _receiver) {
-                console.error(`NOTE: Property ${String(prop)} was accessed on ElementInternals, but this property is not implemented.
-Testing components with ElementInternals is fully supported in e2e tests.`);
-            },
-        });
+      return { setFormValue: (value) => {}};
     }
     constructor(ownerDocument, nodeName, namespaceURI = null) {
         super(ownerDocument, 1 /* NODE_TYPES.ELEMENT_NODE */, typeof nodeName === 'string' ? nodeName : null, null);
